debezium-server-voyager/debezium-server-voyagerexporter/src/main/java/io/debezium/server/ybexporter/EventQueue.java

```diff
@@ -281,6 +252,11 @@ public void warmUp(Map<Long, Long> totalEventsPerSegment) {
                 String line;
                 BufferedReader input;
                 try {
+                    // If the queue segment file has been deleted by the archiver, move to the next
```

Comment:
1. Can we not use checkIfQueueSegmentHasBeenArchivedOrDeleted here as well? Want to avoid a case where there is a bug because of which we conclude that the file does not exist and move on. 
2. There is still an edge-case here where the archiver deletes the file we are reading the file here to fill our dedup cache. Since it's a rare case, I think it's okay... But pls add a comment here explaining this case.

Reply:
1) Doing that check will also skip files that have just been archived and not deleted. So we should stick to the current code.
2) Done